spec/factories/screening.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :screening do
+    movie_id { 1 }
+    hall_id { 1 }
```

Comment:
Never hardcode record id's, this will break
```suggestion
    movie
    hall
```